src/pages/Meet.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+
+.meetContainer {
+    display: flex;
+    flex-direction: column;
+}
+
+.meetUpperSection {
+    background-color: white;
+    padding-top: 5%;
+    margin-bottom: 8%;
+}
+
+.meetMiddleSection {
+    background-color: #c4c4c4;
+}
+
+.meetLowerSection {
+    background-color: white;
+}
+
+.meetContainer h1 {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 108px;
+    line-height: 67px;
+/* identical to box height */
+    text-align: center;
+    color: #000000;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/2747462/99161598-0be4ff80-26c2-11eb-93c2-8244f83244ae.png)
Running into this issue with the default zoom on my browser, adjust the line-height to solve this

Reply:
Thank you! The formatting issues should be fixed now